### ISSUE TITLE
For a multilingual website with an inner folder set as navigation root, the logic for the Contents breadcrumb is incorect

### ISSUE THREAD

**Issue by tiberiuichim** [plone/volto#5699]
If I have the following content tree structure:

```
/en/navroot/something
```

the whole logic in this component collapses, because the store breadcrumbs.items would be just a single item, rooted in the navigation root.

https://github.com/plone/volto/blob/f92880ee4e4c3b88c7cea18c717dd30ecb1e61fa/packages/volto/src/components/manage/Contents/ContentsBreadcrumbs.jsx#L61

**Comment by anshuman-rai-27** [comment]
can you plz explaiin to reproduce this issue ?

**Comment by gawandeabhishek** [comment]
"Hey, @tiberiuichim! I'm interested in addressing this issue, but I'm having trouble understanding your perspective. Could you please provide a detailed explanation? This will help me grasp the problem better and work towards a solution."

**Comment by gawandeabhishek** [comment]
"I'm new to Plone, but I'm passionate about ReactJS and eager to contribute to issue resolution. Could you please explain how I should approach solving this? This would mark my first pull request in React. Thank you!"

@tiberiuichim

**Comment by ichim-david** [comment]
@gawandeabhishek if the author of the bug report needs to explain the bug and how to fix it, it means that you are not ready to fix this issue. If you are curious about learning about the framework, there is documentation available, training videos on Youtube and of course the source course at your disposal for studying.

There is very little to gain in trying to fix bugs for a product that you haven't worked with and you don't know.
If you want to learn something there are tutorials that you can follow and toy projects that you can build but this is a real product with real bugs that need to be fixed by real people that know how to fix them.

**Comment by tiberiuichim** [comment]
In my case, I fixed it by:

- making a physical-breadcrumbs endpoint, https://github.com/eea/eea.climateadapt.plone/commit/ae977c947a1aa2a02e68f84b890b652d1272d073
- customize the ContentsBreadcrumb: https://github.com/eea/volto-cca-policy/commit/a0592aa17b24eb51c8ee146d337094a6cfd37646

**Comment by stevepiercy** [comment]
Test of locking a conversation, and limiting conversation only to people with write access.

**Comment by erral** [comment]
Wouldn't be this fixed if we use the `@navroot` information to build the breadcrumbs?

**Comment by tiberiuichim** [comment]
@erral no, as far as I could see the navroot gives you the content of the ancestor INavigationRoot item (correct me if I'm wrong). The `@breadcrumbs` endpoint already has a `root` field with the path of the navigation root.

I've discussed this with @sneridagh and they had a customized ContentsBreadcrumbs that uses `subsite` information, probably to fix this exact problem.
